src/library/export/engineprimeexportjob.cpp

```diff
@@ -177,10 +167,10 @@ void exportMetadata(djinterop::database* pDatabase,
     snapshot.average_loudness = pTrack->getReplayGain().getRatio();
 
     // Set main cue-point.
-    double cuePlayPos = pTrack->getCuePoint().getPosition();
-    double cueSampleOffset = playPosToSampleOffset(cuePlayPos);
-    snapshot.default_main_cue = cueSampleOffset;
-    snapshot.adjusted_main_cue = cueSampleOffset;
+    mixxx::audio::FramePos cuePlayPos = pTrack->getMainCuePosition();
+    // FIXME: What if the cuePlayPos is invalid?
```

Comment:
@mr-smidge Please verify and handle these cases after we merged this PR.

Reply:
Under what circumstances can the main cue position be invalid?  If not set?

I suppose the simplest solution is just to set to zero/start-of-track in the export.